src/main/webapp/app/entities/modeling-exercise/modeling-exercise-detail.component.html

```diff
@@ -45,13 +45,7 @@ <h2><span jhiTranslate="artemisApp.modelingExercise.detail.title">Modeling Exerc
 
     <button type="submit" (click)="previousState()" class="btn btn-info"><fa-icon icon="arrow-left"></fa-icon>&nbsp;<span jhiTranslate="entity.action.back"> Back</span></button>
 
-    <button
-        type="button"
-        [routerLink]="['/', { outlets: { popup: 'modeling-exercise/' + modelingExercise.id + '/edit' } }]"
-        replaceUrl="true"
-        queryParamsHandling="merge"
-        class="btn btn-primary"
-    >
+    <button type="button" [routerLink]="['/modeling-exercise', modelingExercise.id, 'edit']" replaceUrl="true" queryParamsHandling="merge" class="btn btn-primary">
```

Comment:
```suggestion
    <button type="button" [routerLink]="['/modeling-exercise', modelingExercise.id, '/edit']" replaceUrl="true" queryParamsHandling="merge" class="btn btn-primary">
```

Reply:
Otherwise the link will not work:

'modeling-exercise807/edit'

Reply:
Since I am building this as an array and don't concatenate it, `routerLink` adds slashes where needed, so this works

Reply:
Should we do this consistent with #782?